src/__stories__/components/Colors.tsx

```diff
@@ -18,20 +21,17 @@ const Colors = () => {
           </Typography>
           <div style={{ display: 'flex', gap: 24, marginTop: 16 }}>
             <div style={{ display: 'flex', flexDirection: 'column', gap: 16 }}>
-              {Object.keys(
-                (theme.colors as Record<string, Record<string, string>>)[
-                  sentiment
-                ],
-              )
+              {Object.keys(theme.colors[sentiment as keyof typeof theme.colors])
                 .filter(context => context.includes('background'))
                 ?.map(context => (
                   <BorderedBox
                     key={context}
                     style={{
                       alignItems: 'center',
-                      backgroundColor: (
-                        theme.colors as Record<string, Record<string, string>>
-                      )[sentiment][context],
+                      backgroundColor:
```

Comment:
Why not typing it just once?

```
      {(Object.keys(theme.colors) as Color[]).map(sentiment => {
        const colorContextKeys = Object.keys(
          theme.colors[sentiment],
        ) as AvailableContexts[]

```
as suggested in https://github.com/scaleway/scaleway-ui/pull/1280/files#r828183432